conf/core/ConfigOptions.asciidoc

```diff
@@ -3225,14 +3225,13 @@ Sets that maximum angle that is still considered a power line match. Units in de
 
 A quantile value from 0.0 to 1.0 for the QuantileAggregator.
 
-=== progress.reporting.format
+=== progress.var.print.length.max
 
-* Data Type: string
-* Default Value: ``
+* Data Type: int
+* Default Value: `30`
 
-Sets the progress reporting format to the type specified for classes that support progress
-reporting.  Currently, 'json' is the only valid reporting format. If left blank, progress is not
-reported.
+Maximum length, in characters, allowed when logging a variable value during progress logging
+("--status"). Not honored by all progress logging.
 
```

Comment:
Removed this, as it wasn't used anywhere and we only need a plain text format for now.